### ISSUE TITLE
unable to install starterkit

### ISSUE THREAD

**Issue by dominixdesign** [pattern-lab/patternlab-php#365]
I'm not able to install pattern lab 2 with the starterkit-mustache-demo.

Installation of the starterkit-mustache-demo fails with a fatal error:
`PHP Fatal error:  Uncaught exception 'Alchemy\Zippy\Exception\RuntimeException' with message 'Unable to create the inflator' in`.

According to the guys by the Zippy extenstion, this issue is fixed since 2014.
https://github.com/alchemy-fr/Zippy/issues/69

Any ideas what I'm doing wrong?

I try to get it work on a windows machine.
PHP Version: 5.5.35
Composer: 1.1.2

**Comment by dmolsen** [comment]
@dominixdesign -

First off, thanks for trying it out. Second, I appreciate you trying to do some research on this. I haven't run into this problem but if I could:
- get a more detailed error and
- get you to agree to be a guinea pig to test

I'd appreciate it.

To get the more detailed error try:

```
composer create-project pattern-lab/edition-mustache-standard patternlab2-example -v
```

If you can paste it to gist or here that would be awesome. That will give a stack track that I can work with a little better. 

To be a guinea pig and test changes I make without my having to push full releases you can do the following:

```
composer create-project pattern-lab/edition-mustache-standard:dev-dev patternlab2-test
```

Then when I push changes you'll be able to do a `composer update pattern-lab/core` and get my latest commits.

I'll try to look at this problem in the meantime. Thanks.

**Comment by dmolsen** [comment]
Ah... I think I might know the problem. Zippy is set to a really old version in Core's `composer.json`. I'm going to muck with that. Sorry.

**Comment by dmolsen** [comment]
Welp, that definitely wasn't it. Got my hopes when someone mentioned it was fixed in `2.1` but they really meant `0.2.1`. Core includes `^0.3`. FWIW, your fallback can be "download the zip yourself and put the files from `./dist` in `./source`." I'll document that and try to catch that error.

**Comment by dmolsen** [comment]
This error is now catched as of `v2.7.0` of core. I don't have a great fix but at least the error explains the workaround. Please run `composer update` to get the latest version of PL. [Full list of fixes](https://github.com/pattern-lab/patternlab-php-core/releases/tag/v2.7.0).
